docs/connections.md

```diff
@@ -155,6 +155,11 @@ Then `pip install cx_Oracle`.
 }
 ```
 
+### Oracle User permissions to run the validator tool:
+* GRANT CREATE SESSION TO DVT-USER (Create Session Privilege)
```

Comment:
I wonder if it would be better to be consistent with the BigQuery section and simply list the privileges we require?
For example:

### Oracle user permissions required to run this validator tool:
* CREATE SESSION
* READ or SELECT on any tables to be validated

Just a suggestion, not an essential change (I just like consistency :-) )

Reply:
@nj1973  - Incorporated.